### ISSUE TITLE
Hero Labs import error with Power Profiles active

### ISSUE THREAD

**Issue by KillerGryphon** [Zakarik/foundry-mm3#51]
When importing a Hero Labs profile, a sheet (shared in the Green Ronin channel in the foundry discord) was not able to be correctly imported, as abilities, skills, and defenses were all missing, and powers and advantages had their points at the wrong values. 

Thanks to @devioushearts we discovered that having the Power Profiles option active in the hero configuration was what caused the issue. We were then able to successfully import after toggling that option off.

**Comment by DeviousHearts** [comment]
Yep. I sent you some info on this in our chat, Zakarik but  we found an issue with the Herolab importer. I isolated it to just being when, for some reason, when in Hero Lab if the character is saved with "Power Profiles" selected under "Configure Hero" , it will not transfer the data completely. It failed to transfer over stats, skills, and defenses with zeroes across the board and several advantages had there value reduced to zero as well. Simply turning off "Power Profile" under "Configure Hero" and then saving the XML allowed for a full, normal import.
